integration/class-jetpack.php

```diff
@@ -68,4 +73,44 @@ public static function add_comment_types( $comment_types ) {
 
 		return array_unique( \array_merge( $comment_types, Comment::get_comment_type_slugs() ) );
 	}
+
+	/**
+	 * Add a "Reader" link to the bulk actions dropdown on the following list screen.
+	 *
+	 * @param array $actions The bulk actions.
+	 * @param array $item    The current following item.
+	 *
+	 * @return array The bulk actions with the "Reader" link.
+	 */
+	public static function add_reader_link( $actions, $item ) {
+		// Do not show the link for pending follow requests.
+		if ( 'pending' === $item['status'] ) {
+			return $actions;
+		}
+
+		$reader_id = \get_post_meta( $item['id'], '_activitypub_actor_feed', true );
+
+		if ( $reader_id ) {
+			$url = sprintf( 'https://wordpress.com/reader/feed/%d', (int) $reader_id );
+		} else {
+			$url = sprintf( 'https://wordpress.com/reader/feeds/lookup/%s', $item['identifier'] );
+		}
+
+		$actions['reader'] = sprintf(
+			'<a href="%s" target="_blank">%s</a>',
+			esc_url( $url ),
+			esc_html__( 'View Feed', 'activitypub' )
+		);
+
+		return $actions;
```

Comment:
`View Feed` should probably the first action. Maybe `View` could be enough? Essentially it's their "profile" in the Reader.

<img width="976" height="245" alt="Image" src="https://github.com/user-attachments/assets/708c38ff-bc55-470f-a96e-8f2bae0eaa54" />